content/client-lib-development-guide/features.textile

```diff
@@ -189,7 +189,6 @@ h3(#rest-channel). Channel
 ** @(RSL1d)@ Indicates an error if the message was not successfully published to Ably
 ** @(RSL1e)@ Allows @name@ and or @data@ to be @null@.  If any of the values are @null@, then key is not sent to Ably i.e. a payload with a @null@ value for @data@ would be sent as follows @{ "name": "click" }@
 ** @(RSL1f)@ Unidentified clients using "Basic Auth":https://en.wikipedia.org/wiki/Basic_access_authentication (i.e. any @clientId@ is permitted as no @clientId@ specified):
-** @(RSL1g)@ If the total size of the message or (if publishing an array) messages, calculated per "TO3l8":#TO3l8, exceeds the @maxMessageSize@, then the client library should reject the publish and indicate an error
```

Comment:
@tomczoink I don't follow.  You've just deleted the spec item which has now removed the requirement from the client library spec.  Does this spec item (and the one below) exist in the spec somewhere else?  If not, you need to add this item back in and make sure the spec IDs are unique.  cc @SimonWoolf as I think you added these spec items.